packages/transport-memory/test/transport-memory.test.js

```diff
@@ -0,0 +1,8 @@
+/**
+ * Created by arolave on 14/05/2017.
+ */
+xdescribe('no tests', () => {
+  xit('no test', () => {
+    
```

Comment:
yeah, I told you about it. for now we use it only to test the metrics.. if we will expand our usage, we will add tests. I can't see a point to do it now